### PR TITLE
Implement getting a backtrace from a Windows Kernel Dump for x64

### PR DESCRIPTION
**Pull request by GustavoLCR**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

* Implements getting a backtrace from a windows kernel dump for x64 targets
* Fixes `ompb` command not setting the priority of maps
* Add PE header to mappings

**Test plan**

Tests pass

**Closing issues**

None

### CODE REVIEW

File: librz/debug/p/debug_dmp.c [hunk 1]
```diff
@@ -391,13 +406,39 @@ static RzList *rz_debug_dmp_threads(RzDebug *dbg, int pid) {
 	return ret;
 }
 
+static RzList *dmp_get_modules(DmpCtx *ctx) {
+	if (ctx->type == DMP_DUMPTYPE_TRIAGE) {
```
Comment (XVilka) [inline librz/debug/p/debug_dmp.c]: Maybe better to reverse this condition?

File: librz/debug/p/native/bt/windows-x64.c [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+}
+
+static inline ut16 read_slot16(RzDebug *dbg, ut64 unwind_code_offset, int *index) {
+	ut8 buf[2];
+	READ_AT(unwind_code_offset + *index * sizeof(ut16), buf, sizeof(buf));
+	(*index)++;
+	return rz_read_le16(&buf);
+}
+
+static bool backtrace_windows_x64_internal(RZ_IN RzDebug *dbg, RZ_INOUT RzList **out_frames, RZ_INOUT struct context_type_amd64 *context) {
```
Comment (XVilka) [inline librz/debug/p/native/bt/windows-x64.c]: I recommend to split this function into smaller.

**Review by XVilka — CHANGES_REQUESTED** [review]

File: librz/debug/p/debug_dmp.c [hunk 3]
```diff
@@ -425,6 +466,69 @@ static bool rz_debug_dmp_kill(RzDebug *dbg, int pid, int tid, int sig) {
...
+
+RzList *rz_debug_dmp_frames(RzDebug *dbg, ut64 at) {
+	RzCore *core = dbg->corebind.core;
+	DmpCtx *ctx = dbg->plugin_data;
+	RzList *ret = NULL;
+	if (ctx->windctx.is_arm) {
+		RZ_LOG_DEBUG("TODO\n");
+		if (ctx->windctx.is_64bit) {
+		} else {
+		}
```
Comment (wargio) [inline librz/debug/p/debug_dmp.c]: maybe comment here?

File: librz/debug/p/debug_dmp.c [hunk 4]
```diff
@@ -425,6 +466,69 @@ static bool rz_debug_dmp_kill(RzDebug *dbg, int pid, int tid, int sig) {
...
+				}
+				dbg->corebind.applyBinInfo(core, file, RZ_CORE_BIN_ACC_MAPS | RZ_CORE_BIN_ACC_SYMBOLS);
+				dbg->corebind.cmdf(dbg->corebind.core, "idp %s", pdbpath);
+				dbg->corebind.cmdf(dbg->corebind.core, "ompb %d", ((RzBinFile *)ctx->bf)->id);
+				free(exepath);
+				free(pdbpath);
+			}
+			rz_list_free(modules);
+		} else {
+			RZ_LOG_DEBUG("TODO\n");
```
Comment (wargio) [inline librz/debug/p/debug_dmp.c]: same?

**Review by XVilka — APPROVED** [review]

**Pull request merged**